### ISSUE TITLE
Full screen does not show navigation bar nor tabs

### ISSUE THREAD

**Issue by Sh1d0w** [brave/brave-browser#4331]
## Description 
When you make Brave fullscreen on macOS by clicking the green arrow at the top left right next to the minimize button, Brave shows only the content of the webpage, but cuts out all tabs and the address bar. It is impossible to reveal them even if you CMD+L to focus the address bar and start typing.

Chrome on other hand properly shows tabs and address bar in fullscreen mode.

## Brave version (brave://version info)
Version 0.63.55 Chromium: 74.0.3729.131 (Official Build) (64-bit)

**Comment by rebron** [comment]
@Sh1d0w Can you post a screenshot? I'm not able to reproduce on macOS. We were able to reproduce this on Windows and Linux.

**Comment by Sh1d0w** [comment]
@rebron Even better, I've made a video:

https://i.imgur.com/CbXLEO8.mp4

**Comment by bsclifton** [comment]
@Sh1d0w that is the behavior we get on Windows and Linux. What version of macOS are you running?

**Comment by Sh1d0w** [comment]
@bsclifton 
<img width="698" alt="Screenshot 2019-05-13 at 9 39 15" src="https://user-images.githubusercontent.com/5074917/57600618-005fc800-7563-11e9-9f93-c6d1fd2a72b5.png">

**Comment by morbec** [comment]
I've the same issue using the Nightly build. Version 0.67.41 Chromium: 75.0.3770.38 (Official Build) nightly (64-bit).
macOS Mojave (10.14.5).

**Comment by srirambv** [comment]
Actually reproduced on macOS 10.14.5. It shows the same behaviour similar to Linux/Windows

**Comment by AlexWang-16** [comment]
I can confirm this is also happening on macOS Catalina 10.15.1

**Comment by rebron** [comment]
@Sh1d0w and @AlexWang-16 There's a setting "Always show toolbar in full screen" under View. Can you guys select that and see if that works for you?

**Comment by AlexWang-16** [comment]
Yes, that's what I was looking for. The description of this option was not obvious to me. Thanks for your help!

**Comment by PragmaticMeru** [comment]
Solution for windows? doesn't appear that option here.

**Comment by santos7117** [comment]
@rebron 
I can't find that option "Always show toolbar in full screen"

**Comment by santos7117** [comment]
> @Sh1d0w and @AlexWang-16 There's a setting "Always show toolbar in full screen" under View in the menu at the top. Can you guys select that and see if that works for you?

may be this is not available in the current version

**Comment by SpBills** [comment]
Is there any solution for windows/linux users?

**Comment by fgouvea10** [comment]
Also happening on macOS Sonoma version 14.2.1

**Comment by prijwal** [comment]
@rebron I always select the the option "Always show toolbar in full screen", but somehow may be in a month, it gets reset to its default state which is off  :( , do you have any idea why ?

**Comment by mjg-foundation** [comment]
I can't find this setting anywhere. I found "toggle fullscreen toolbar" and set a shortcut, but it does nothing.

**Comment by heyaco** [comment]
man the devs at brave are fukin slow. ui/ux incompetance up n down the chain
